### ISSUE TITLE
Cache the U2F private key in storage

### ISSUE THREAD

**Issue by saleemrashid** [trezor/trezor-mcu#251]
Since U2F can be used without the PIN, we should cache the U2F private key so that we do not perform operations on the mnemonic until the device is unlocked.

**Comment by saleemrashid** [comment]
I think we should postpone this until after #253 is fixed because I don't want to add more fields to the `Storage` structure.
